### PR TITLE
Feature/tao 5239 brs integration

### PR DESCRIPTION
**Pull request by krampstudio**
Add a multiple delete action, to work with the resource selector.
Add it also through the controller to setup ACLs

### CODE REVIEW

File: actions/structures.xml [hunk 1]
```diff
@@ -30,19 +30,10 @@
...
-                        <icon id="icon-find"/>
-                    </action>
                     <action id="item-class-new" name="New class" url="/taoItems/Items/addSubClass" context="resource" group="tree" binding="subClass">
                         <icon id="icon-folder-open"/>
                     </action>
-                    <action id="item-delete" name="Delete" url="/taoItems/Items/deleteItem" context="instance" group="tree" binding="removeNode" >
-                        <icon id="icon-bin"/>
-                    </action>
-                    <action id="item-class-delete" name="Delete" url="/taoItems/Items/deleteClass" context="class" group="tree" binding="removeNode" >
+                    <action id="item-delete-all" name="Delete" url="/taoItems/Items/deleteAll" context="resource" multiple="true" group="tree" binding="removeNodes">
```
Comment (jsconan) [inline actions/structures.xml]: Is the remove of `item-delete` action intentional? It is still linked above (https://github.com/oat-sa/extension-tao-item/pull/294/files#diff-384f0ccf46984d5475abe29b5b92c56cR17).

**Comment by krampstudio** [comment]
That was a mistake.

File: actions/structures.xml [hunk 2]
```diff
@@ -45,6 +39,9 @@
                     <action id="item-class-delete" name="Delete" url="/taoItems/Items/deleteClass" context="class" group="tree" binding="removeNode" >
                         <icon id="icon-bin"/>
                     </action>
+                    <action id="item-delete-all" name="Delete" url="/taoItems/Items/deleteAll" context="resource" multiple="true" group="tree" binding="removeNodes">
```
Comment (jsconan) [inline actions/structures.xml]: The display name of 'delete all' is the same as for the 'delete one'.

Comment (krampstudio) [reply actions/structures.xml]: They aren't supposed to be displayed at the same time, multiple and single contexts are exclusives. The bug is fixed in tao-core

**Pull request merged**